apps/avifdec.c

```diff
@@ -155,7 +157,7 @@ int main(int argc, char * argv[])
                 returnCode = 1;
             }
         } else {
-            fprintf(stderr, "Unrecognized file extension: %s\n", inputFilename);
+            fprintf(stderr, "Unrecognized file extension: %s\n", outputFilename);
             returnCode = 1;
```

Comment:
If we get here, the output file extension is ".avif". I am not sure if ""Unrecognized file extension" is the most appropriate error message. But at least this makes avifdec match the error handling in avifenc.